nikola/plugins/task/categories.py

```diff
@@ -0,0 +1,172 @@
+    show_list_as_subcategories_list = False
+    generate_atom_feeds_for_post_lists = True
+    template_for_classification_overview = "tags.tmpl"
+    add_postcount_in_overview = True
+    always_disable_rss = False
+    apply_to_posts = True
+    apply_to_pages = False
+    minimum_post_count_per_classification_in_overview = 1
+    omit_empty_classifications = True
+    also_create_classifications_from_other_languages = True
+
+    def set_site(self, site):
+        """Set site, which is a Nikola instance."""
+        super(ClassifyCategories, self).set_site(site)
+        self.show_list_as_index = self.site.config['CATEGORY_PAGES_ARE_INDEXES']
+        self.template_for_single_list = "tagindex.tmpl" if self.show_list_as_index else "tag.tmpl"
+
+    def is_enabled(self, lang=None):
+        """Return True if this taxonomy is enabled, or False otherwise."""
+        return True
+
+    def classify(self, post, lang):
+        """Classify the given post for the given language."""
+        cat = post.meta('category', lang=lang).strip()
+        return [cat] if cat else []
+
+    def get_classification_friendly_name(self, classification, lang, only_last_component=False):
+        """Extract a friendly name from the classification."""
+        classification = self.extract_hierarchy(classification)
+        return classification[-1] if classification else ''
+
+    def get_overview_path(self, lang, type='page'):
+        """A path handler for the list of all classifications."""
+        if self.site.config['CATEGORIES_INDEX_PATH'][lang]:
+            return [_f for _f in [self.site.config['CATEGORIES_INDEX_PATH'][lang]] if _f], 'never'
+        else:
+            return [_f for _f in [self.site.config['CATEGORY_PATH'][lang]] if _f], 'always'
+
+    def slugify_tag_name(self, name, lang):
+        """Slugify a tag name."""
+        if lang is None:  # TODO: remove in v8
+            utils.LOGGER.warn("ClassifyCategories.slugify_tag_name() called without language!")
+            lang = ''
+        if self.site.config['SLUG_TAG_PATH']:
+            name = utils.slugify(name, lang)
+        return name
+
+    def slugify_category_name(self, path, lang):
+        """Slugify a category name."""
+        if lang is None:  # TODO: remove in v8
+            utils.LOGGER.warn("ClassifyCategories.slugify_category_name() called without language!")
+            lang = ''
+        if self.site.config['CATEGORY_OUTPUT_FLAT_HIERARCHY']:
+            path = path[-1:]  # only the leaf
+        result = [self.slugify_tag_name(part, lang) for part in path]
+        result[0] = self.site.config['CATEGORY_PREFIX'] + result[0]
+        if not self.site.config['PRETTY_URLS']:
+            result = ['-'.join(result)]
+        return result
+
+    def get_path(self, classification, lang, type='page'):
+        """A path handler for the given classification."""
+        return [_f for _f in [self.site.config['CATEGORY_PATH'][lang]] if
```

Comment:
Changed. Do you want parantheses for all similar tuple returns? There are some more of them.